tests.py

```
@@ -1,32 +1,34 @@
+
+from kinesis import Consumer, MemoryCheckPointer, Producer, RedisCheckPointer, exceptions
 from kinesis.aggregators import (
     Aggregator,
     KPLAggregator,
-    SimpleAggregator,
-    NewlineAggregator,
     ListAggregator,
     NetstringAggregator,
+    NewlineAggregator,
     OutputItem,
+    SimpleAggregator,
```

Ruff (0.11.9)

`kinesis.aggregators.SimpleAggregator` imported but unused

Remove unused import: `kinesis.aggregators.SimpleAggregator`

(F401)